nightwatch.conf.js

```diff
@@ -0,0 +1,37 @@
+const os = require('os');
+
+module.exports = {
+  src_folders: [
+    'test/e2e', // Where you are storing your Nightwatch e2e/UAT tests
+  ],
+  output_folder: './reports', // reports (test outcome) output by nightwatch
+  selenium: { // downloaded by selenium-download module (see readme)
+    start_process: true, // tells nightwatch to start/stop the selenium process
+    server_path: os.homedir() + '/.selenium-binaries/selenium/2.53.1/selenium-server-standalone-2.53.1.jar',
```

Comment:
see note below...I'd recommend fixing the version number so you don't have to update this all the time.